plugin/testdata/router-path-test.lua

```diff
@@ -0,0 +1,16 @@
+local typedefs = require "kong.db.schema.typedefs"
+
+local schema = {
+  name = "path-test",
```

Comment:
Let's keep the plugin name consistent with the filename. I understand that there is the alias, but it isn't an exact alias of `router_path`. You should be able to update the function signature to pass in the plugin name or to use the basename of the filename.

```suggestion
  name = "route-path-test",
```

Reply:
Alrighty.